app/src/main/java/org/fcitx/fcitx5/android/input/candidates/floating/LabeledCandidateItemUi.kt

```diff
@@ -27,8 +27,8 @@ class LabeledCandidateItemUi(
     }
 
     fun update(candidate: FcitxEvent.Candidate, active: Boolean) {
-        val fg = if (active) theme.genericActiveForegroundColor else theme.keyTextColor
-        val altFg = if (active) theme.genericActiveForegroundColor else theme.altKeyTextColor
+        val fg = if (active) theme.genericActiveForegroundColor else theme.candidateLabelColor
```

Comment:
Just use `candidateLabelColor` and `candidateTextColor` on "label" and "text" respectively

```suggestion
        val labelFg = if (active) theme.genericActiveForegroundColor else theme.candidateLabelColor
        val fg = if (active) theme.genericActiveForegroundColor else theme.candidateTextColor
```